MCprep_addon/spawner/mcmodel.py

```diff
@@ -208,29 +217,29 @@ def read_model(context, model_filepath):
 			elif os.path.isfile(base_path):
 				elements, textures = read_model(context, base_path)
 			else:
-				env.log("Failed to find mcmodel file " + parent_filepath)
+				env.log(f"Failed to find mcmodel file {parent_filepath}")
 
-	current_elements = obj_data.get("elements")
+	current_elements: Element = obj_data.get("elements")
 	if current_elements is not None:
 		elements = current_elements  # overwrites any elements from parents
 
-	current_textures = obj_data.get("textures")
+	current_textures: Texture = obj_data.get("textures")
 	if current_textures is not None:
 		if textures is None:
 			textures = current_textures
 		else:
 			for img in current_textures:
 				textures[img] = current_textures[img]
 
-	env.log("\nfile:" + str(model_filepath), vv_only=True)
+	env.log(f"\nfile: {model_filepath}", vv_only=True)
 	# env.log("parent:" + str(parent))
 	# env.log("elements:" + str(elements))
 	# env.log("textures:" + str(textures))
 
 	return elements, textures
 
 
-def add_model(model_filepath, obj_name="MinecraftModel"):
+def add_model(model_filepath: Path, obj_name: str="MinecraftModel") -> Object:
```

Comment:
To prevent confusion with the Python `Object` type
```suggestion
def add_model(model_filepath: Path, obj_name: str="MinecraftModel") -> bpt.types.Object:
```